From c861eae83bae3116d330efb3c6061e2de4fdcbce Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Sun, 5 Apr 2015 22:26:12 +0300
Subject: [PATCH] tools: make_rel_symlink.py can also use python2

The make_rel_symlink.py script is compatible with both python2 and python3.
Don't hard code a requirement for python3.

Patch status: sent upstream

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
---
 tools/make_rel_symlink.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/make_rel_symlink.py b/tools/make_rel_symlink.py
index 896637ff93e0..ff0403a0397a 100755
--- a/tools/make_rel_symlink.py
+++ b/tools/make_rel_symlink.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
 
 import os
 import os.path
-- 
2.1.4

